src/_includes/docs/install/reqs/macos/set-path.md

```diff
@@ -0,0 +1,39 @@
+{% assign terminal=include.terminal %}
+{% assign target = include.target %}
+
+### Add Flutter to your `PATH`
+{:.no_toc}
+
+To run Flutter commands in {{terminal}},
+You need to add Flutter to the `PATH` environment variable.
+THis section covers the permanent addition of the path to the
+Flutter to your `PATH` environment variable.
```

Comment:
```suggestion
you need to add Flutter to the `PATH` environment variable.
You can add it manually each time you launch a terminal 
but it's easier, as shown here,
to add it to your environment so that it's always available.
```

Reply:
I'd like to keep this more opinionated and avoid writing about another method that they'll find they need to change later. That's what made the original macOS install page so cumbersome.

Reply:
I disagree, in this doc it's so opinionated I find it difficult to understand what is required to use Flutter and what's just recommended.  Like why `.zshenv` and not `.zshrc`?  Why would Flutter need me to update `GEM_HOME`?  Have we now messed up the environment for someone who already has Ruby installed?  Why are we in the business of telling people to update that env?

Reply:
The `zsh` docs say that environment variables should be set in `.zshenv` because that file is loaded regardless of it being a login or an interactive shell. The other files are not always loaded. 

This is also what opinionated means in this case. We're not offering options, we're providing a path to completion. When this is published, we may get issues filed against this and we can address any raised concerns then.

Reply:
I guess I'm disagreeing that this doc should be opinionated.  I suggest we put this page back to:
1. Install Xcode (link to Xcode in the Mac App Store, or Apple docs)
2. Install Cocoapods (link to CocoaPods install page)
3. Put Flutter on your PATH (link to how to put things on your PATH, or maybe a blog post or something like "here's an example of how to do that!"

etc.

As someone who has had to close many open issues and add workarounds to the Flutter tool related to installation and environment issues resulting from our _own_ docs, I'd really prefer not to be in the business of keeping the dependency installation docs up-to-date, or to confuse experienced developers who know how to add executables to their PATH, or make it seem like we have dependencies that we don't (Homebrew).

Once it's installed and built, the `flutter` tool itself has more up-to-date hints about how to get your environment set up (you have Xcode installed, but you need to accept the license, etc)